pkg/catalog/template.go

```diff
@@ -46,7 +46,7 @@ const NewPackageTemplate = `
 ![]({{CreateSequenceDiagram $appName $e}})
 
 ### Request types
-{{if eq (len $e.Param) 0}}
+{{if and (eq (len $e.Param) 0) (eq $e.RestParams nil) }}
```

Comment:
is this tested?
in go templating you can't equate a pointer type to nil: https://goplay.space/#QOWlt3VKeQr

`nil` is casted to `false`, same as any other zero type

Reply:
Great catch, changed it to `with` instead of `if`